examples/fxa-geodb-example.js

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var path = require('path');
```

Comment:
PROTIP: You can add `/* eslint-disable strict, no-console */` after the license there and it will disable those 2 rules for this entire file. Then you can lint the examples/* folder too.

Conversely, you can add a nested examples/.eslintrc file which just turns those 2 rules off (and will still inherit the parent rules):

```yaml
rules:
  no-console: 0
  strict: 0
```